src/api/initialDataTypes.js

```diff
@@ -74,53 +74,197 @@ export type AvailableVideoChatProviders = $ReadOnly<{|
+    +[site_name: string]: {|
+      name: string,
+      text: string,
+      hint: string,
+      url_pattern: string,
+    |},
+  |},
   realm_default_language: string,
-  realm_default_twenty_four_hour_time: boolean,
+
+  // TODO(server-5.0): Replaced in feat. 99 by
+  // realm_user_settings_defaults.twenty_four_hour_time; there, only present
+  // if realm_user_settings_defaults is given in fetch_event_types
+  realm_default_twenty_four_hour_time?: boolean,
+
+  // TODO(server-5.0): Added in feat. 101, replacing realm_allow_message_deleting
+  realm_delete_own_message_policy?: number,
+
   realm_description: string,
+  realm_digest_emails_enabled: boolean,
+  realm_digest_weekday: number,
   realm_disallow_disposable_email_addresses: boolean,
+
+  // TODO(server-5.0): Added in feat. 75, replacing realm_allow_community_topic_editing
+  realm_edit_topic_policy?: number,
+
+  realm_email_address_visibility: number,
   realm_email_auth_enabled: boolean,
   realm_email_changes_disabled: boolean,
+  realm_emails_restricted_to_domains: boolean,
+
+  // TODO(server-4.0): Added in feat. 55.
+  realm_giphy_rating?: number,
+
+  // TODO: Do servers send this, here or in realm-update events?
   realm_google_hangouts_domain: string,
+
   realm_icon_source: string,
   realm_icon_url: string,
   realm_inline_image_preview: boolean,
   realm_inline_url_embed_preview: boolean,
-  realm_invite_by_admins_only: boolean,
+
+  // TODO(server-4.0): Replaced in feat. 50 by realm_invite_to_realm_policy
+  realm_invite_by_admins_only?: boolean,
+
   realm_invite_required: boolean,
+
+  // TODO(server-4.0): Added in feat. 50, replacing realm_invite_by_admins_only
+  realm_invite_to_realm_policy?: number,
+
+  realm_invite_to_stream_policy: number,
   realm_is_zephyr_mirror_realm: boolean,
+  realm_logo_source: string,
+  realm_logo_url: string,
   realm_mandatory_topics: boolean,
+  realm_message_content_allowed_in_email_notifications: boolean,
 
   // In 5.0 (feature level 100), the representation the server sends for "no
   // limit" changed from 0 to `null`, and 0 became an invalid value. (For
   // the invalid-value part, see zulip/zulip#20131.)
   realm_message_content_delete_limit_seconds: number | null,
 
   realm_message_content_edit_limit_seconds: number,
+
+  // TODO(server-3.0): Special value `null` replaced with -1 in feat. 22
   realm_message_retention_days: ?number,
+
+  // TODO(server-4.0): Added in feat. 56
+  realm_move_messages_between_streams_policy?: number,
+
   realm_name: string,
   realm_name_changes_disabled: boolean,
+  realm_night_logo_source: string,
+  realm_night_logo_url: string,
   realm_notifications_stream_id: number,
   realm_password_auth_enabled: boolean,
+  realm_plan_type: number,
   realm_presence_disabled: boolean,
+  realm_private_message_policy: number,
+  realm_push_notifications_enabled: boolean,
+
+  // TODO: Do servers send this, here or in realm-update events?
   realm_restricted_to_domain: boolean,
```

Comment:
Hmm, yeah. I wonder if this was an old name for `realm_emails_restricted_to_domains` (or for an old version of that.)